packages/fxa-content-server/app/scripts/views/support.js

```diff
@@ -157,8 +157,10 @@ const SupportView = BaseView.extend({
 
     if (this.supportForm.isValid()) {
       this.submitBtn.removeClass('disabled');
+      this.submitBtn.prop('disabled', false);
     } else {
       this.submitBtn.addClass('disabled');
+      this.submitBtn.prop('disabled', true);
```

Comment:
If the `disabled` class name is only being used for styling I think we can remove the addClass/removeClass usage because buttons with the `disabled` prop [also applies](https://github.com/mozilla/fxa/blob/master/packages/fxa-content-server/app/styles/modules/_button-row.scss#L83) that styling, I think 🤔

Reply:
I *think* it is only for styling. I will try it locally to see if the prop alone applies the same styling.

Reply:
It is only for styling. I have updated it :) Thanks so much for the suggestion